recipes/inspectds/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "inspectds" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/inspectds-{{ version }}.tar.gz
+  sha256: 8e7cdb3bd52fa98793497507da23bd2b91d41e2f2baa5f2561be10a709d19048
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win or py<=38]
```

Comment:
This looks like pure Python...is there any reason you need to skip Windows?

Reply:
The build was failing and I don't know how to make it pass. E.g. 
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=696689&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf